main_test.go

```diff
@@ -1799,7 +1799,7 @@ filter_policy {
 			MongoClient: mongoClient,
 		},
 		Registry: registry,
-		Logger:   logrus.NewEntry(logger),
+		Logger:   rondlogrus.NewLogger(logger),
```

Comment:
can't we use the noop logger here?

Reply:
Yes, we could use but in the main we use logrus so I think it's better for the test